cmd/nerdctl/container/container_create_linux_test.go

```diff
@@ -174,3 +180,72 @@ func TestCreateWithTty(t *testing.T) {
 	withTtyContainer := base.InspectContainer(withTtyContainerName)
 	assert.Equal(base.T, 0, withTtyContainer.State.ExitCode)
 }
+
+// TestIssue2993 tests https://github.com/containerd/nerdctl/issues/2993
+func TestIssue2993(t *testing.T) {
```

Comment:
You need to mark this test as DockerIncompatible.

Reply:
Thanks !
Added `testutil.DockerIncompatible(t)` to tests.